### PR TITLE
Added geoviews RGB type

### PR DESCRIPTION
**Pull request by philippjfr**
As the title says, adds a geo/crs-aware RGB Element type. Addresses https://github.com/ioam/geoviews/issues/67

### CODE REVIEW

File: geoviews/element/geo.py [hunk 1]
```diff
@@ -214,6 +215,28 @@ class Image(_Element, HvImage):
     group = param.String(default='Image')
 
 
+class RGB(_Element, HvRGB):
```
Comment (jlstevens) [inline geoviews/element/geo.py]: Couldn't ``_Element`` have been called ``CartElement`` or similar? It has ``_abstract=True`` so I don't see the need for the underscore...

Comment (philippjfr) [inline geoviews/element/geo.py]: It could have been yes, but at this point I'll just leave it.

Comment (jlstevens) [inline geoviews/element/geo.py]: Ok, maybe file an issue for next time.

**Comment by jlstevens** [comment]
Looks fine to me. Merging.

**Pull request merged**